libtiledbsoma/src/cli/arrow_adapter.h

```diff
@@ -1,7 +1,9 @@
 #ifndef ARROW_ADAPTER_H
 #define ARROW_ADAPTER_H
 
-#include <tiledbsoma/tiledbsoma>
+#include <tiledb/tiledb>
+#include "../utils/column_buffer.h"
+#include "../utils/logger.h"
 #ifndef ARROW_SCHEMA_AND_ARRAY_DEFINED
 #include "carrow.h"
 #endif
```

Comment:
I have learned more recently from the Arrow docs that "they" use header guard `ARROW_C_DATA_INTERFACE`.  The idea (by them) is if one protects with that one should be fine.  We could try that, possibly inside `carrow.h`.